ports/pravila00-enum-string/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "pravila00-enum-string",
+  "version-date": "2023-16-10",
```

```suggestion
  "version-date": "2023-10-16",
```
Note the format：
https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/users/versioning.md#version-date